### PR TITLE
Show hide unknown

### PR DESCRIPTION
**Pull request by dsamojlenko**
# Description

Show/hide "Why is the prognosis unknown?" field based on radio selection 

Also includes a change to how Vue is loaded - ~~this change will require changes in #72~~ changes made in #72 to accommodate

# Test Instructions

- Navigate to add_condition
- Under 'Is your patient's condition likely to', select Unknown

### CODE REVIEW

**Review by jeffmaher — DISMISSED** [review]
Tested in Firefox. LGTM. 🍕

**Review by jeffmaher — DISMISSED** [review]

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
I'm not seeing anything but localization keys in the PR, is that expected?

File: routes/add_condition/add_condition.njk [hunk 1]
```diff
@@ -11,6 +11,10 @@
 
     {{ radioButtons('condition_outlook', { '1':'add_condition.condition_outlook_improve','2':'add_condition.condition_outlook_deteriorate', '3':'add_condition.condition_outlook_staysame', '4':'add_condition.condition_outlook_unknown'}, data.condition_outlook, 'add_condition.condition_outlook', errors, { required: true }) }}
 
+    <div id="unknown_details" class="ml-4">
+        {{ textArea('condition_outlook_unknown', 'add_condition.prognosis_unknown') }}
```
Comment (CalvinRodo) [inline routes/add_condition/add_condition.njk]: This should have `{ required: true }` since it's required when it's displayed.

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
Need to add validation for the unknown field, it needs to be required when the item is selected in the radio box.

**Comment by dsamojlenko** [comment]
Added validation for all fields

**Review by CalvinRodo — CHANGES_REQUESTED** [review]
When submitting a page with nothing entered it displays the unknown field even if nothing is selected. No errors are associated with the field: 

Screen Shot 2019-11-25 at 10 53 11 AM

**Comment by dsamojlenko** [comment]
> When submitting a page with nothing entered it displays the unknown field even if nothing is selected. No errors are associated with the field:
> 
> Screen Shot 2019-11-25 at 10 53 11 AM

This issue will be resolved with an upstream change coming later today: https://github.com/cds-snc/node-starter-app/pull/133

**Review by jeffmaher — APPROVED** [review]
Works! 🍕

**Review by CalvinRodo — APPROVED** [review]

**Pull request merged**